.test/great_expectations/config/config_variables.yml

```diff
@@ -0,0 +1,38 @@
+# This config file supports variable substitution which enables: 1) keeping
+# secrets out of source control & 2) environment-based configuration changes
+# such as staging vs prod.
+#
+# When GE encounters substitution syntax (like `my_key: ${my_value}` or
+# `my_key: $my_value`) in the great_expectations.yml file, it will attempt
+# to replace the value of `my_key` with the value from an environment
+# variable `my_value` or a corresponding key read from this config file,
+# which is defined through the `config_variables_file_path`.
+# Environment variables take precedence over variables defined here.
+#
+# Substitution values defined here can be a simple (non-nested) value,
+# nested value such as a dictionary, or an environment variable (i.e. ${ENV_VAR})
+#
+#
+# https://docs.greatexpectations.io/en/latest/how_to_guides/configuring_data_contexts/how_to_use_a_yaml_file_or_environment_variables_to_populate_credentials.html
+
+instance_id: 9a5bf883-42c2-49c3-9128-a707efea14d3
+redshift:
+  drivername: postgresql+psycopg2
+  host: sp-com-snplow-eng-aws-prod1-redshift-cluster.cdzqkzl74pqa.eu-central-1.redshift.amazonaws.com
```

Comment:
Remove hardcoded URL

Reply:
Actually this whole file shouldn't have been committed. I must have accidentally done so when I rebased. :s